config/rabbitmq.js

```diff
@@ -2,10 +2,24 @@
 
 const config = require('config');
 
+/**
+ * convert value to Boolean
+ * @method convertToBool
+ * @param {(Boolean|String)} value
+ * @return {Boolean}
+ */
+function convertToBool(value) {
+    if (typeof value === 'boolean') {
+        return value;
+    }
+
+    return String(value).toLowerCase() === 'true';
+}
+
 const rabbitmqConfig = config.get('scheduler').rabbitmq;
 const { protocol, username, password, host, port, exchange, vhost, connectOptions } = rabbitmqConfig;
 const amqpURI = `${protocol}://${username}:${password}@${host}:${port}${vhost}`;
-const schedulerMode = config.get('scheduler').enabled;
+const schedulerMode = convertToBool(config.get('scheduler').enabled);
```

Comment:
Is this not boolean?
https://github.com/screwdriver-cd/queue-service/blob/master/config/default.yaml#L186

Reply:
Yes, the default value you showed is boolean.
But when we change this value, we need to set value as an environment variable.
We use Kubernetes as a build cluster, and Kubernetes can not set boolean value as an environment variable.
We need to set string value like "true".